device/mellanox/x86_64-nvidia_sn5600-r0/Mellanox-SN5600-C256S1/buffers_defaults_objects.j2

```diff
@@ -150,18 +155,13 @@
     "BUFFER_QUEUE": {
 {% if dynamic_mode is not defined %}
 {% for port in port_names_active.split(',') %}
-        "{{ port }}|0": {
+        "{{ port }}|0-3": {
             "profile" : "q_lossy_profile"
         },
 {% endfor %}
-{% for port in port_names_active.split(',') %}
-        "{{ port }}|1-3": {
-            "profile" : "q_lossy_minus_7_profile"
```

Comment:
It seems that q_lossy_minus_7_profile is not called by other functions,  should we remove it too?

Reply:
it doesn't impact functionality but can confuse. fixed